src/main/java/com/exactpro/th2/common/schema/factory/AbstractCommonFactory.java

```diff
@@ -658,6 +670,14 @@ protected ConnectionManager getRabbitMqConnectionManager() {
         });
     }
 
+    public MessageIdBuilderImpl getMessageIdBuilder() {
+        return new MessageIdBuilderImpl();
+    }
+
+    public EventIdBuilderImpl getEventIdBuilder() {
+        return new EventIdBuilderImpl();
+    }
+
```

Comment:
MessageIdBuilder and EventIdBuilder interfaces are not final. Let's make methods similar to the next instead
```
public MessageIDBuilder newMessageIDBuilder() {
    return MessageID.newBuilder()
                        .setBookName(boxConfiguration().getBookName)
}
```

Reply:
MessageIdBuilderImpl and EventIdBuilderImpl classes are redundant for the current issue

Reply:
done